src/CommunityToolkit.Maui/Alerts/Snackbar/SnackBar.shared.cs

```diff
@@ -136,4 +143,28 @@ void OnDismissed()
 		IsShown = false;
 		_weakEventManager.HandleEvent(this, EventArgs.Empty, nameof(Dismissed));
 	}
+}
+
+/// <summary>
+/// Extension methods for <see cref="VisualElement"/>.
+/// </summary>
+public static class VisualElementExtension
```

Comment:
let's move it back to VisualElementExtensions. We may call Toast and other Anchored Controls from it

Reply:
I prefer to keep it in `Snackbar` for two reasons:
- It  keeps the extension method close to the `Snackbar` code
  - In XCT, we had a few times where a bug was fixed in the base code, but the extension method was forgotten about.
- `VisualElementExtensions` becomes a dumping ground
  - Because `VisualElement` encompasses so many things in MAUI, it quickly became a huge file of 100s of lines of code